src/interfaces/IPlugin.sol

```diff
@@ -98,6 +101,11 @@ struct PluginManifest {
 }
 
 interface IPlugin {
+    /// @notice Retrieves the address of the VersionRegistry contract associated with this plugin.
+    /// @dev This function can be used by external contracts to verify the VersionRegistry.
+    /// @return The address of the VersionRegistry contract.
+    function getVersionRegistry() external view returns (address);
```

Comment:
I think this function is not necessary if version registry address is included in PluginManifest, how do you think about this?

Reply:
Removed in c2815022d5122175d69f041500e27359dd79fd43